fuel-tests/Cargo.toml

```diff
@@ -23,6 +23,7 @@ fuel-crypto = { version = "0.4", features = ["random"] }
 fuel-gql-client = { path = "../fuel-client", features = ["test-helpers"] }
 fuel-storage = "0.1"
 fuel-tx = { version = "0.9", features = ["serde-types", "builder", "internals"] }
+fuel-txpool = { path = "../fuel-txpool" }
```

Comment:
This needs a version number, no?

Reply:
this is a local unpublished crate, so it doesn't need version numbers for any deps coming from this repo

Reply:
Ah right.